lib/galaxy/webapps/galaxy/controllers/admin.py

```diff
@@ -185,7 +189,7 @@ def get_value(self, trans, grid, role):
     # Grid definition
     title = "Roles"
     model_class = model.Role
-    default_sort_key = "name"
+    default_sort_key = "-update_time"
```

Comment:
I think sorting by name for groups and roles is what admins would expect as default.

Reply:
This made it easier when creating or editing Roles/Groups, since they appear at the top of the list while working with them.

Reply:
I don't it's so common to modify a role/group you just created. Most of the time I need to add new users to an existing group when they join the corresponding research group at random times.

Reply:
Alright, I restored the previous sort order. Let me know what you think of the remaining changes. Thanks for looking into this.

Reply:
Thanks!